en/docs/install-and-setup/setup/api-controller/managing-microgateways/managing-microgateways-with-ctl.md

```diff
@@ -0,0 +1,298 @@
+# Managing Microgateways
+
+WSO2 API Controller (apictl) can be used as a developer CLI tool for Microgateway. It enables interacting with one or more Microgateways without having to perform the following tasks via the WSO2 API Manager (WSO2 API-M).
+
+- Deploy an API
+- View the list of deployed APIs
+- Undeploy an API
+
+!!! info
+    **Before you begin** 
+
+    -  Download the Microgateway distribution from the [github release page's](https://github.com/wso2/product-microgateway/releases) and follow the Quick Start Guide attached with the release.
+
+    -  Make sure the apictl is downloaded and initialized, if not follow the steps in [Download and Initialize the CTL Tool]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#download-and-initialize-the-ctl-tool). Following the guide upto [Check the version of the CTL]({{base_path}}/install-and-setup/setup/api-controller/getting-started-with-wso2-api-controller/#check-the-version-of-the-ctl) will be sufficient to continue. 
+
+
+## Add a Microgateway Environment
+
+A microgateway here refers to the entire cluster which includes all three components: adapter, router and enforcer. By adding a microgateway environment, the url to the adapter will be stored in a config file, and therefore enables login. 
+
+Once you have added a microgateway environment, and logged in, other commands such as `apictl mg deploy api` can be used by only specifying an environment. This allows you to use such commands without entering the microgateway adapter url, username, and password every time.
+
+-   **Command**
+    ```bash
+    apictl mg add env <environment-name> --adapter <adapter-url> -k
```

Comment:
All the commands on this page seem to have `-k` used, which is the optional insecure mode. Any reason we are doing this? Ideally we should not be giving too much prominence to `-k`

Reply:
+1 to remove -k if there is not any specific reason. We have removed -k from the other commands as well recently via https://github.com/wso2/docs-apim/pull/3252/commits/d300d2302b90a49ddf20081cd30e28c4df1768d0

Reply:
No special reason. I hadn't added the cert to apictl while testing. Removing -k from all commands and adding a step to include the cert.